tests/fixtures/success.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I will need to spend sometime with the UTs, since I tried to use this but something didn't work in my local. For now this fixture is good to be kept as it contains a valid file, it's just the bats that didn't work.

I'll figure out in a follow-up